apollo-runtime/build.gradle

```diff
@@ -32,11 +32,8 @@ android {
 }
 
 dependencies {
-  compile dep.jsr305
-  compile dep.supportAnnotations
-  compile dep.okHttp
-  compile dep.moshi
-  provided project(":apollo-api")
```

Comment:
Yeah, this is mainly needed for the test fixtures (since they need to add those as well). I'll look for an alternative.